lib/ts/Layout.cc

```diff
@@ -41,134 +41,96 @@ Layout::get()
 }
 
 void
-Layout::create(const char *prefix)
+Layout::create(ts::string_view const &prefix)
```

Comment:
We had a long discussion on this (which rat holed in to comparative compiler argument passing techniques) but you said we should be consistent and other places `string_view` is passed by value.